oeso.owl

```diff
@@ -361,6 +327,44 @@ The Object Ontology is licensed by CC 4.0 (http://creativecommons.org/licenses/b
     
 
 
+    <!-- http://www.opensilex.org/vocabulary/oeso#isAdministrativeContactOf -->
+
+    <owl:ObjectProperty rdf:about="http://www.opensilex.org/vocabulary/oeso#isAdministrativeContactOf">
```

Comment:
for isAdministrativeContactOf, isCoordinatorOf, isScientificContactOf, isScientificSupervisorOf you should use the inverse property to have 'has*' pattern and have Agent on domain.

Reply:
I used thoses because we decided it last time together